hack/machineset.sh

```diff
@@ -173,6 +173,74 @@ $(get_spec $infraID $az $provider)
 EOF
 }
 
+# get_vsphere_ms creates a MachineSet for vSphere Cloud Provider
+get_vsphere_ms() {
+
+  if [ "$#" -lt 2 ]; then
+    error-exit incorrect parameter count for get_vsphere_ms $#
+  fi
+
+  local infraID=$1
+  local provider=$2
+
+  # set golden image template name
+  # TODO: read from parameter
+  template="windows-golden-images/windows-server-2004-template"
+
+  # get vsphere providerSpec for any worker machine
+  # TODO: filter machines in running phase
```

Comment:
The real TODO is to make just one `oc get machines` call and just pass the data around.  This is the 3rd call being introduced and can be avoided. PS: not asking for that change here.